packages/build/src/core/normalize_flags.js

```diff
@@ -62,6 +63,7 @@ const computeTelemetry = function (flags, envOpts) {
 }
 
 const REQUIRE_MODE = 'require'
+const DEFAULT_EDGE_HANDLERS_DIST = '.netlify/edge-handlers-dist/'
```

Comment:
We should also add TypeScript types:

https://github.com/netlify/build/blob/02afafba375d3ae7d2a4f4b0796f49abc5bc27cb/packages/build/types/netlify_plugin_constants.d.ts#L24

https://github.com/netlify/build/blob/02afafba375d3ae7d2a4f4b0796f49abc5bc27cb/packages/build/test-d/netlify_plugin_options.ts#L11

Reply:
Ah, great catch! Added in https://github.com/netlify/build/pull/4221/commits/95cb67408ea80d64c105e0e5c691012ce53f145a.